src/io/flutter/run/FlutterDebugProcess.java

```diff
@@ -15,12 +15,12 @@
 import com.intellij.ui.GuiUtils;
 import com.intellij.ui.content.Content;
 import com.intellij.xdebugger.XDebugSession;
-import com.jetbrains.lang.dart.ide.runner.server.vmService.DartVmServiceDebugProcessZ;
 import com.jetbrains.lang.dart.util.DartUrlResolver;
 import io.flutter.actions.ReloadFlutterApp;
 import io.flutter.actions.RestartFlutterApp;
 import io.flutter.run.daemon.FlutterApp;
 import io.flutter.run.daemon.RunMode;
+import io.flutter.server.vmService.DartVmServiceDebugProcessZ;
```

Comment:
If we are not going to merge into the Dart plugin then we could drop the Z on this class name. I think I added it to distinguish it from the superclass in a way that would invite merging (but I'm not certain about that). Not that we need to worry about that in this PR; I'm just mentioning it for consideration later.

Reply:
Sounds good. Do you think it would make sense to merge this class directly into DartVmServiceDebugProcess or to give it a Flutter specific name?
Naively merging seems like it would make sense in which case it should be a separate CL to keep this CL as just a simple set of renames and copies.

Reply:
Merging in a separate PR would be best IMO.